_vector-search/ai-search/hybrid-search/inner-hits.md

```diff
@@ -0,0 +1,648 @@
+The response includes the matched parent documents along with the relevant nested `inner_hits` for both the `user` and `location` nested fields. Each inner hit shows which nested object matched and how strongly it contributed to the overall hybrid score:
+
+```json
+...
+{
+  "hits": [
+    {
+      "_index": "my-nlp-index",
+      "_id": "1",
+      "_score": 1.0,
+      "inner_hits": {
+        "location": {
+          "hits": {
+            "max_score": 0.44583148,
+            "hits": [
+              {
+                "_nested": {
+                  "field": "location",
+                  "offset": 1
+                },
+                "_score": 0.44583148,
+                "_source": {
+                  "city": "Udaipur",
+                  "state": "Rajasthan"
+                }
+              }
+            ]
+          }
+        },
+        "user": {
+          "hits": {
+            "max_score": 0.4394061,
+            "hits": [
+              {
+                "_nested": {
+                  "field": "user",
+                  "offset": 0
+                },
+                "_score": 0.4394061,
+                "_source": {
+                  "name": "John Alder",
+                  "age": 35
+                }
+              }
+            ]
+          }
+        }
+      }
+      // Additional details omitted for brevity
+    },
+    {
+      "_index": "my-nlp-index",
+      "_id": "2",
+      "_score": 5.0E-4,
+      "inner_hits": {
+        "user": {
+          "hits": {
+            "max_score": 0.31506687,
+            "hits": [
+              {
+                "_nested": {
+                  "field": "user",
+                  "offset": 0
+                },
+                "_score": 0.31506687,
+                "_source": {
+                  "name": "John Wick",
+                  "age": 46
+                }
+              },
+              {
+                "_nested": {
+                  "field": "user",
+                  "offset": 1
+                },
+                "_score": 0.31506687,
+                "_source": {
+                  "name": "John Snow",
+                  "age": 40
+                }
+              }
+            ]
+          }
+        }
+        // Additional details omitted for brevity
+      }
+    }
+  ]
+  // Additional details omitted for brevity
+}
+...
+```
+
+## Using the explain parameter
+
+To understand how inner hits contribute to the hybrid score, you can enable explanation. The response will include detailed scoring information. For more information about using `explain` with hybrid queries, see [Hybrid search explain]({{site.url}}{{site.baseurl}}/vector-search/ai-search/hybrid-search/explain/).
+
+`explain` is an expensive operation in terms of both resources and time. For production clusters, we recommend using it sparingly for the purpose of troubleshooting.
+{: .warning}
+
+First, add the `hybrid_score_explanation` processor to the search pipeline you created in Step 2:
```

Comment:
Should we link to Step 2?

Reply:
Not necessary because the query is repeated here.